src/connector.js

```diff
@@ -48,7 +48,6 @@ if ( lang.startsWith( "fr" ) ) {
 
 paramsDetect.isContextSearch = !winPath.endsWith( '/sr/srb.html' ) && !winPath.endsWith( '/sr/sra.html' );
 paramsDetect.isAdvancedSearch = !!document.getElementById( 'advseacon1' ) || winPath.endsWith( '/advanced-search.html' ) || winPath.endsWith( '/recherche-avancee.html' );
-paramsDetect.enableHistoryPush = !paramsDetect.isAdvancedSearch;
```

Comment:
Please bring this back

Reply:
For pagination (as well as query summary), please use a custom event called `searchEvent`